### PR TITLE
suggest expand on str.split

### LINKED ISSUE #20: split with expand
`series.str.split(',',expand=True)`

### PR DESCRIPTION
**Pull request by DeanLa**
# closes #20

- [ ] closes #

# Description

### CODE REVIEW

File: dovpanda/core.py [hunk 1]
```diff
@@ -206,3 +206,20 @@ def dont_append_with_loop(arguments):
...
                     'and then <code>pd.concat(list_of_dfs)</code> in one go')
+
+
+@ledger.add_hint('Series.str.split', 'post')
+def suggest_expand(res, arguments):
+    expand = arguments.get('expand')
+    pat = arguments.get('pat')
+    if expand:
+        return
+    if hasattr(res, 'name'):
```
Comment (galbraun) [inline dovpanda/core.py]: I'm not sure what you want to check here..if that for the case of series by itself the `name` property will still exists but with Null value, no? i think this condition will never be false

File: dovpanda/core.py [hunk 2]
```diff
@@ -206,3 +206,20 @@ def dont_append_with_loop(arguments):
...
+        col = res.name
+    else:
+        col = 'series'
+    if not expand:
```
Comment (galbraun) [inline dovpanda/core.py]: This condition is redundant, you verified expand is false before

Comment (tzurE) [reply dovpanda/core.py]: +1

File: dovpanda/core.py (hunk 1 above)
Comment (DeanLa) [reply dovpanda/core.py]: I'm trying to be defensive. Maybe `res` returns as some weird datatype on some weird edge cases.

Comment (galbraun) [reply dovpanda/core.py]: So it won't be a series, please change the col name to something more generic than series. Maybe a `structure`?

Comment (DeanLa) [reply dovpanda/core.py]: column seems fitting as the user's main use case is to do the op on column

**Pull request merged**